tests/acceptance/features/apiSpaces/changingFilesShare.feature

```diff
@@ -0,0 +1,40 @@
+@api @skipOnOcV10
+Feature:
+
+  Background:
+    Given using spaces DAV path
+    And these users have been created with default attributes and without skeleton files:
+      | username |
+      | Alice    |
+      | Brian    |
+
+
+  @issue-4421
+  Scenario: Move files between shares by different users
```

Comment:
Should this scenario be be in `moveSpaces.feature`?

Reply:
IMO, the scenario is more related to changing received share rather moving between spaces.